### PR TITLE
Emergency party kit

### PR DESCRIPTION
**Pull request by mkanke-real**
## About the PR

This update adds a new container for the mime and clown. If medics get all these kits to heal physical harm, service deserves one to heal the soul. This adds the basics of the kit to the game that will have its contents updated after I make a party popper and maybe a different pastry to go in it. PR will follow to add it to clown/mime rooms as a default item.
## Why / Balance

Its just fun and adds a less suspect way for more nefarious clowns/mimes to bring items around the station.
## Technical details

Adds a basic storage along with its sprite.
## Requirements

- [ X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- add: Adds emergency party kit

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Fun/EmergencyPartyKit.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+- type: entity
+  id: EmergencyPartyKit
+  parent: BaseItem
+  name: emergency party kit
+  description: For when you need a party STAT and cant wait for cargo.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/EmergencyPartyKit.rsi
+    heldPrefix: EmergencyPartyKit
```
Comment (marc-pelletier) [inline Resources/Prototypes/Entities/Objects/Fun/EmergencyPartyKit.yml:9]: I'm not great with YAML but I've seen that `heldPrefix` is generally under `Item` types, whereas under `Sprite` types you want to use `state`

**Comment by taydeo** [comment]
You wanna add more stuff mk?

**Comment by mkanke-real** [comment]
> You wanna add more stuff mk?

Currently when you try to open the menu no inventory screen pops up so I still need to debug that. Also still deciding between some sprites as I am not really satisfied right now with the one thats on there

**Comment by taydeo** [comment]
Hi, some issues.

All custom files need to be in a _Funkystation folder.

**Comment by mkanke-real** [comment]
> Hi, some issues.
> 
> All custom files need to be in a _Funkystation folder.

All moved over now

**Pull request merged**